dt-assets/js/advanced-search.js

```diff
@@ -0,0 +1,181 @@
+        results_total.html(new_global_hits_count);
+
+        // Update section offset value
+        evt.currentTarget.parentNode.parentNode.querySelector("#advanced-search-modal-results-table-row-section-head-hidden-offset").setAttribute("value", results[0]['offset']);
+
+        // Insert latest finds...!
+        results[0]['posts'].forEach(function (post) {
+          current_section_head.closest('tr').after(build_result_table_row(post)).next('tr').slideDown('fast');
+        });
+      } else {
+        // Hide more search option when there are no further hits to be returned.
+        evt.currentTarget.style.display = 'none';
+      }
+
+    }).catch(error => {
+      console.log(error);
+    });
+  }
+
+  function reset_widgets() {
+    $('.advanced-search-modal-form-input').val('');
+    $('.advanced-search-modal-results-div').slideUp('fast');
+    $('.advanced-search-modal-results').html('').fadeOut('fast');
+  }
+
+  function execute_search_query() {
+    let query = $('.advanced-search-modal-form-input').val();
+    let selected_post_type = $('input[name=advanced-search-modal-post-types]:checked').val();
+
+    if (query.trim() === "") {
+      return;
+    }
+
+    // Dispatch search query and display api response accordingly
+    $('.advanced-search-modal-results-div').slideDown('fast', function (data) {
+      let spinner = '<span class="loading-spinner active"></span>';
+      $('.advanced-search-modal-results').html(spinner).fadeIn('slow', function () {
+        rest_api.advanced_search(encodeURI(query), selected_post_type, 0).then(api_data => {
+          display_results(api_data, function () {
+            $('.advanced-search-modal-results').fadeIn('fast');
+          });
+        }).catch(error => {
+          console.log(error);
+        });
+      });
+    });
+  }
+
+  function display_results(api_data, callback) {
+    let results = api_data['hits'];
+    let total_hits = api_data['total_hits'];
+    let results_html = "";
+
+    // Update global hits count
+    $('.advanced-search-modal-results-total').html(total_hits);
+
+    // Iterate through results, displaying accordingly
+    results_html += '<table class="advanced-search-modal-results-table" style="border: none;"><tbody style="border: none;">';
+    results.forEach(function (result) {
+
+      results_html += '<tr style="background: #f5f5f5; padding: 0px;">';
+      results_html += '<td style="text-align: left; padding: 10px;"><a style="margin: 0px; padding: 2px 5px 2px 5px; max-height: 20px; min-height: 20px;" class="button hollow advanced-search-modal-results-table-row-section-head-load-more">' + fetch_more_text + '</a></td>';
+      results_html += '<td style="text-align: right;">';
+      results_html += '<b>' + result['post_type'] + '</b></td>';
+      results_html += '<input type="hidden" id="advanced-search-modal-results-table-row-section-head-hidden-offset" value="' + result['offset'] + '">';
```

Comment:
please escape any variable that is to be outputed in html
`window.lodash.escape()`